### PR TITLE
Banners: replace random.randint with random.choice

### PR DESCRIPTION
**Pull request by GuilhermeHideki**
random.choice uses a generator, removing the need of some if/else chains and keeping the index value

### CODE REVIEW

**Comment by GuilhermeHideki** [comment]
WDYT @shirosaidev?

File: diskover.py [hunk 1]
```diff
@@ -43,84 +43,68 @@
...
+""",
+)
+
 
-    """ % (color, version)
+def print_banner(version):
+    """
+    Prints a random banner.
+
+    :param str version: The program version
```
Comment (shirosaidev) [inline diskover.py:104]: remove this and extra spaces in comment

File: diskover.py [hunk 2]
```diff
@@ -43,84 +43,68 @@
...
+    """
+    color = random.choice(BANNER_COLORS)
```
Comment (shirosaidev) [inline diskover.py:106]: change random.choice to just choice

File: diskover.py [hunk 3]
```diff
@@ -15,7 +15,7 @@
 from rq import SimpleWorker, Queue
 from rq.registry import StartedJobRegistry
 from datetime import datetime
-from random import randint
+import random
```
Comment (shirosaidev) [inline diskover.py:18]: change to from random import choice

**Comment by GuilhermeHideki** [comment]
@shirosaidev I took the liberty of marking the changes resolved because they looked simple, but let me know if they are not correct

**Pull request closed without merging**